src/dbnode/persist/fs/commitlog/writer.go

```diff
@@ -129,6 +130,7 @@ func newCommitLogWriter(
 		sizeBuffer:         make([]byte, binary.MaxVarintLen64),
 		seen:               bitset.NewBitSet(defaultBitSetLength),
 		logEncoder:         msgpack.NewEncoder(),
+		logEncoderBuff:     make([]byte, 0, 16384),
```

Comment:
Nit: make size a const (I know this was me hah).

Reply:
Haha will fix